dashboard/app/views/reports/search_for_teachers.html.haml

```diff
@@ -0,0 +1,27 @@
+- content_for(:head) do
+  = stylesheet_link_tag    'jquery.handsontable.full', media: 'all'
+  = javascript_include_tag 'jquery.handsontable.full'
+%h1 Search for Teachers
+
+%h3
+  Address Filter
+  %input#filter
+%br/
+#teacherTable
+
+:javascript
+  var baseFilter = '#{ params[:filter]}';
+  var data = eval(#{ data.try(:to_json)});
```

Comment:
why is this eval here? Would `var data = #{ data.try(:to_json)};` not be the same thing without, y'know, using an eval?